src/test/groovy/org/assertj/generator/gradle/SimpleBuild.groovy

```diff
@@ -116,14 +116,13 @@ class SimpleBuild {
                 mavenCentral()
             }
                         
-            dependencies {
-                // https://mvnrepository.com/artifact/com.google.guava/guava
-                compile group: 'com.google.guava', name: 'guava', version: '20.0'
+            dependencies { 
+                implementation group: 'javax.annotation', name: 'javax.annotation-api', version: '1.3.2'
                 
-                // https://mvnrepository.com/artifact/org.assertj/assertj-core
-                testCompile group: 'org.assertj', name: 'assertj-core', version: '3.8.0'
+                implementation group: 'com.google.guava', name: 'guava', version: '30.0-jre'
                 
-                testCompile group: 'junit', name: 'junit', version: '4.12'
+                testImplementation group: 'org.assertj', name: 'assertj-core', version: '3.22.0'
```

Comment:
```suggestion
                testImplementation group: 'org.assertj', name: 'assertj-core', version: '3.24.2'
```

Reply:
I intentionally matched the versions used in the generator itself: https://github.com/assertj/assertj-assertions-generator/blob/main/pom.xml

And proceeded to get them wrong, I've updated them accordingly though.